public/js/monitor.js

```diff
@@ -336,6 +336,80 @@ function recruitmentLogic() {
   });
 }
 
+function addWaitlistSignupButtonListeners() {
+  document.querySelectorAll(".relay-sign-up-btn").forEach(btn => {
+
+    btn.addEventListener("click", async(e) => {
+      const relayEndpoint = new URL("/join-waitlist", document.body.dataset.serverUrl);
+      const signUpCallout = document.querySelector(".relay-sign-up");
+
+      signUpCallout.classList.add("sending-email");
+      try {
+        const response = await fetch(relayEndpoint, {
+          headers: {
+            "Content-Type": "application/json; charset=utf-8",
+          },
+          mode: "same-origin",
+          method: "POST",
+          body: JSON.stringify({"emailToAdd": "add-user-email"}),
+        });
+        if (response && response.status === 200) {
+          setTimeout(()=> {
+            signUpCallout.classList.add("email-sent");
+            signUpCallout.classList.remove("sending-email");
+          }, 500);
+        }
+      } catch(e) {
+        // we need error messaging
+      }
+    });
+  });
+}
+
+function addWaitlistObservers() {
+
+  const privateRelayCtas = document.querySelectorAll(".private-relay-cta");
+
+  if (privateRelayCtas.length === 0) {
+    return;
+  }
+  const availableIntersectionObserver = ("IntersectionObserver" in window);
```

Comment:
I had a question about adding IntersectionObserver from a [browser compat](https://github.com/mozilla/blurts-server/blame/f80d96d09b4d6dc2bdb997c9af1612a84a37f5e1/public/js/resolve-breaches.js#L51) stand point, but looks like we have had it [on the site for a while](https://github.com/mozilla/blurts-server/blame/f80d96d09b4d6dc2bdb997c9af1612a84a37f5e1/public/js/resolve-breaches.js#L51) without issue? 

Can you confirm that Monitor does not support browsers that don't support this? (Namely, IE11)

Reply:
Confirm.